reporter-opentsdb/src/test/java/io/ultrabrew/metrics/reporters/opentsdb/OpenTSDBHttpClientTest.java

```diff
@@ -86,6 +87,17 @@ public void testWriteFailsManualFlush() throws Exception {
     assertEquals(0, client.currentBatchSize);
   }
 
+  @Test
+  public void testNullStringsAndTagsValidation() throws Exception {
+    OpenTSDBHttpClient client = new OpenTSDBHttpClient(DUMMY_DB_URI, 64, true);
+    client.write(null, new String[] { "host", "web01" }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { null, "web01" }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host", null }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host", "web01" }, 1534055562000000003L, null);
+    client.write("cpu_load_short.temp", null, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host" }, 1534055562000000003L, "80");
```

Comment:
Shouldn't this test verify something about the resulting behavior?

Reply:
oh darn lost that bit, will restore it.